src/components/Preview.js

```diff
@@ -219,6 +221,11 @@ class Preview extends Component {
             />
           ))}
         </div>
+        {!numFrames ? (
+          <div className="Preview-no-frames">
+            No frames have been captured...
```

Comment:
Why this message instead? Not nearly as helpful in guiding the user to fix the problem.

Reply:
![Screen Shot 2019-07-03 at 2 17 44 PM](https://user-images.githubusercontent.com/35031893/60625709-6f0c1580-9d9d-11e9-90cf-165ab65019b1.png)
![Screen Shot 2019-07-03 at 2 15 24 PM](https://user-images.githubusercontent.com/35031893/60625715-729f9c80-9d9d-11e9-9bd6-8cfa61157381.png)
If you use the frame editor to delete all frames you have the option to undo delete. The bottom screenshot made more sense to me but I may just be overthinking it. (technically you can "add" a frame by hitting undo delete)

Reply:
but good call, I'll just switch it back

Reply:
done